### ISSUE TITLE
Add a test to audit tool: check for filename correctness in images directory

### ISSUE THREAD

**Issue by TBRC-Travis** [buda-base/asset-manager#127]
We expect images in the "images" folder to conform to our standard naming convention:

`[image_group_id][zero_padded_four_digit_sequence].[filename_extension]`

for example, image group folder **W1KG14-I1KG16** should have images named:

```
I1KG160001.tif
I1KG160002.tif
I1KG160003.jpg
I1KG160004.jpg
...etc
```
We should add a required test to audit tool for this.

**Comment by jimk-bdrc** [comment]
We don't have optional tests in audit tool. We have only tests. A user designates the results as warnings or errors. The way to make a test optional is to put its known errors into the ErrorsAsWarnings list of properties.

**Comment by jimk-bdrc** [comment]
I wonder if a more expansive pattern matching might be also useful. It might take the form of

_parent_property_ : _pattern_ 

Where
parent_property : [ArchiveImageGroupParent | DerivedImageGroupParent ]. (names from the properties shell)
pattern : specifies a substitution pattern: we can use tokens such as 'WorkRID' and ImageGroupRID' and 'Sequence:0n' 'extension'
(case insensitive) where workRid and ImageGroupRID stand for what you think they stand for, and `Sequence:0n' means 'a sequence of _n_ digits, with the 0 (zero) before it representing the fill character. So 'Sequence:.3' means three digits, left filled with '.'

**Comment by TBRC-TimB** [comment]
My understanding is that IGRIDs are assigned when the WIDs are assigned, during ingestion and processing by the Librarians when they enter information into DLMS. We don't process images into archival tifs and then web images until the work has had that initial ingestion.

**Comment by jimk-bdrc** [comment]
Implemented in #167 
Deployed onto sattva
